functorch/_src/aot_autograd.py

```diff
@@ -858,6 +858,8 @@ class AOTConfig:
 
 def aot_dispatch_base(flat_fn, flat_args: List[Tensor], aot_config: AOTConfig):
     fw_module = make_fx(flat_fn, aot_config.decompositions)(*flat_args)
+    fw_module.graph.eliminate_dead_code()
```

Comment:
Yeah, not allowed to do this.

I guess the correct long term fix is to run functionalization on forwards, it's kind of a pain though because most of our stack actually does know how to deal with mutation in the forwards case lol

Reply:
Ok, I do this change because I see the FX module has some redundant codes, changes are removed now.